src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] results = new String[names.length];
+        LocalDate from = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate to = LocalDate.parse(dateTo, FORMATTER);
+        int counter = 0;
+        for (String name : names) {
+            int money = 0;
+            for (String record : data) {
+                String[] arr = record.split(" ");
+                String n = arr[1];
```

Comment:
The variable name 'n' is not informative. According to the checklist, variable names should be descriptive. Consider renaming it to something like 'employeeName'.